syntax = "proto3";

package upvest;

message Product {
  string product_id = 1;
  string name = 2;
  string category = 3;
  int64 created_at = 4;

  // New optional field (backward-compatible)
  string description = 5;  // optional field
}